interfaces/ResizeObserver.idl

```diff
@@ -11,7 +11,7 @@ interface ResizeObserver {
     void disconnect();
 };
 
-callback ResizeObserverCallback = void (sequence<ResizeObserverEntry> entries, ResizeObserver observer);
+callback ResizeObserverCallback = void (sequence<ResizeObserverEntry> entries, ResizeObserver observer)
```

Comment:
Where did the semicolon go?

Reply:
It's been updated in the IDL (https://github.com/WICG/ResizeObserver/pull/56) but not pushed, I think.